### PR TITLE
runcommand.sh: updated doc for %XRES%, %YRES%

### PR DESCRIPTION
**Pull request by Gemba**
As discussed here initially: https://retropie.org.uk/forum/topic/30662/intellivision-jzintv-for-sdl2-proposed-scriptmodule/3

Added some escape sequences for doxygen to keep `%` char in HTML output.

### CODE REVIEW

File: scriptmodules/supplementary/runcommand/runcommand.sh [hunk 1]
```diff
@@ -2,12 +2,15 @@
 
 # This file is part of The RetroPie Project
 #
-# The RetroPie Project is the legal property of its developers, whose names are
-# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+# The RetroPie Project is the legal property of its developers, whose names
+# are too numerous to list here. Please refer to the COPYRIGHT.md file
+# distributed with this source.
```
Comment (joolswills) [inline scriptmodules/supplementary/runcommand/runcommand.sh:7]: This may be best left so it's the same as other sources

**Comment by joolswills** [comment]
Looks ok to me. It's not ideal with the quoting but I had missed that `%` was being dropped. Maybe we can work around this via the bash doxygen code we use so we don't need to quote it. I'll see if there's any alternative solutions but appreciate the work on this.

**Comment by Gemba** [comment]
Whatever will be done to remediate the % situation, at least the documentation is there. :)

**Comment by joolswills** [comment]
Think the quoting is the best way for now so merging. Thanks.

**Comment by joolswills** [comment]
I rebased / squashed the commits and adjusted commit msg slightly. Thanks for the contribution.

**Pull request merged**